### ISSUE TITLE
JS on Pathway page

### ISSUE THREAD

**Issue by ariutta** [wikipathways/wikipathways-upgrade#25]
There are many JS errors happening on the pathway page. I'm not sure whether they're all from the same problem, but here's a list:
- [ ] window.alert: `upgrade.wikipathways.org says: [empty]`
- [x] http://upgrade.wikipathways.org/w/skins/wikipathways/bootstrap.css Failed to load resource: the server responded with a status of 404 (Not Found)
- [x] wikibits.js:619 Uncaught ReferenceError: mediaWiki is not defined
  (anonymous function) @ wikibits.js?303:619 ([Relevant MW documentation](https://www.mediawiki.org/wiki/ResourceLoader/Legacy_JavaScript#wikibits.js))
- [ ] xrefpanel.js:16 Uncaught ReferenceError: wgServer is not defined
- [ ] Pathway:WP2059:194 Uncaught ReferenceError: XrefPanel is not defined (note: this does not always show up. Why?)
- [ ] Cannot read property 'registerTrigger' of undefined

I suspect at least some of these might have to do with the XrefPanel.

**Comment by AlexanderPico** [comment]
Skins dir was renamed per upgrade convention. Should be http://upgrade.wikipathways.org/w/skins/WikiPathways/bootstrap.css.  I've changed the one's I could find and no longer see this error.

**Comment by ariutta** [comment]
@AlexanderPico: where did you make this change? I couldn't find where.

**Comment by ariutta** [comment]
Was it `extensions/LocalHooks.php`?

**Comment by AlexanderPico** [comment]
Yes. For some reason, most of the files in `/extensions/` are NOT in the repo, per the .gitignore. I can't think of a good reason to excluded these from the repo though...  I've seen this in other places to. Not having them in the repo limits searching via github...
